go.mod

```diff
@@ -62,5 +63,7 @@ require (
 replace (
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.1.7-0.20210622111912-ef00f8ac3d76
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+
+	github.com/line/wasmvm => github.com/shiki-tak/wasmvm v0.16.3-testing.0.20220624032404-3e5e9ef10a08
```

Comment:
When the following PRs are merged, the latest version is applied.
https://github.com/line/wasmvm/pull/66